src/notebooks/controllers/vscodeNotebookController.ts

```diff
@@ -245,16 +250,10 @@ export class VSCodeNotebookController implements Disposable, IVSCodeNotebookCont
                 nbDocumentUris.length ? 'and documents ' + nbDocumentUris.join(', ') : ''
             }`
         );
-        if (this.isDisposed) {
-            // If we don't see this again, then we can remove this check and exit this function early.
-            // Else the code needs to be fixed to ensure we don't dispose twice or don't depend on the events again.
```

Comment:
Based on the logs, this was never happening, hence removed these commands the exit from here if we've disposed once before.